src/htm/regions/ClassifierRegion.cpp

```diff
@@ -78,10 +78,12 @@ namespace htm {
     },
     inputs: {
       bucket:  { description: "The quantized value of the current sample, one from each encoder if more than one, for the learn step",
-                           type: Real64,  count: 0},  
-      pattern: { description:  "An SDR output bit pattern for a sample.  Usually the output of the SP or TM. For example: activeCells from TM",  
-                           type: SDR, count: 0} 
-    }, 
+                           type: Real64,  count: 0},
+      pattern: { description:  "An SDR output bit pattern for a sample.  Usually the output of the SP or TM. For example: activeCells from TM",
+                           type: SDR, count: 0},
+      learnPattern: { description:  "An SDR output bit pattern for a sample.  Usually the output of the SP or TM. For example: activeCells from TM",
```

Comment:
how would this learn/infer pattern input work with async data? I mean, Say you want to learn a single pattern, and then make inference on 10 samples. What would be on learnPattern input? And will this behave correctly? 

We might need (already have) a separate `learn` & `infer () const`, and call it separately in the ClassRegion.